config/AcConstants.fpp

```diff
@@ -48,6 +48,12 @@ constant DpWriterNumProcPorts = 5
 @ The size of a file name string
 constant FileNameStringSize = 256
 
+@ The size of a file name in an AssertFatalAdapter event
+@ Note: File names in assertion failures are also truncated by
+@ the constants FW_ASSERT_TEXT_SIZE and FW_LOG_STRING_MAX_SIZE, set
+@ in FpConfig.h.
```

Comment:
This is fantastic - I really appreciate properly documenting this. It'll save a bunch of pain for fprime users that need to raise the assert message size

Reply:
Thanks, Josh!